README.md

```diff
@@ -11,6 +11,7 @@
 1. skipper <https://github.com/stratoscale/skipper>
 1. minikube (for tests)
 1. kubectl
+1. Python modules `pip install waiting`
```

Comment:
Why is this needed?

Reply:
https://github.com/openshift/assisted-service/blob/master/tools/wait_for_assisted_service.py#L3

Reply:
It is installed on https://github.com/openshift/assisted-service/blob/master/Dockerfile.assisted-service-build#L15
Maybe we should split into a `requirements.txt` file that could be used even without the Dockerfile.
Than inside the Dockerfile we should use
```dockerfile
COPY requirements.txt .
python3 -m pip install -r requirements.txt
```

Reply:
I think that is a reasonable approach. For now, I think calling out the need for `waiting` is sufficient for someone to make it through the README.